system/Database/SQLite3/Builder.php

```diff
@@ -78,12 +79,20 @@ protected function _truncate(string $table): string
     protected function _updateBatch(string $table, array $keys, array $values): string
     {
         if (version_compare($this->db->getVersion(), '3.33.0') >= 0) {
-            return parent::_updateBatch($table, $keys, $values);
+            // return parent::_updateBatch($table, $keys, $values);
```

Comment:
Why just commented out?
If this line is no longer needed, entire `if` is not needed.

Reply:
I had this commented out to test it. It should not be commented out.